etc/oracle-admin/partitions.sql

```diff
@@ -0,0 +1,495 @@
+/*
+Objective:
+==========
+	The following PL/SQL code will be used to create 
+	an automatic process to remove 90 days older partitions 
+	in FILETRANSFERSDB and FILEMETADATA tables in production database.
+
+Database: 
+=========
+    These code is applied to the following databases:
+
+	- production:    cms_analysis_reqmgr@cmsr
+    - preproduction: cmsweb_analysis_preprod@int2r
+
+Dependency/Requirements:
+========================
+	1. We must ensure that we have proper permission to create 
```

Comment:
care to say how we check ? WHo is "we" ? the schema owner ?

Reply:
yeah, bad wording, i improved it